internal/builtinfunctions/functions.go

```diff
@@ -159,7 +159,7 @@ func getBooleanToStringFunction() schema.CallableFunction {
 		schema.NewDisplayValue(
 			schema.PointerTo("boolToString"),
 			schema.PointerTo(
-				"Represents `true` as \"true\", and `false` as \"false\".",
+				"Returns `\"true\"` for `true`, and `\"false\"` for `false`.",
```

Comment:
For the other `toString` functions above, you use the terminology "Converts a [...]". Using the same terminology consistently would make it easier to read.

Reply:
I prefer returns here because it explicitly describes all possible states.

Reply:
I'd like to see a consistent template for describing the "this to that" functions, and "returns" works as well as "converts" ... I just dislike using different phrases for similar patterns.

Reply:
There. They're changed to that format.